scripts/deploy_site.sh

```diff
@@ -29,6 +29,6 @@ gcloud beta app deploy \
   --version $deployVersion \
   --no-promote \
   $BASEDIR/../notifier.yaml \
-  $BASEDIR/../app.yaml  \
+  $BASEDIR/../app.staging.yaml  \
```

Comment:
With this change, how would we deploy to prod?
I think we need a conditional in here somewhere, or we need to specify a command-line argument with the name of the yaml file to deploy (or just the 'staging' or 'prod' part of it).